docs/tutorial/part-zero/index.md

```diff
@@ -10,7 +10,7 @@ Before you start to code, you’ll need to familiarize with some core web techno
 It’s not necessary to be an expert with these already — if you’re not, don’t worry! You’ll pick up a lot through the course of this tutorial series; These are some of the main web technologies you’ll use when building a Gatsby site:
 
 - **HTML**: A markup language that every web browser is able to understand. It stands for HyperText Markup Language. HTML gives your web content a universal informational structure, defining things like headings, paragraphs, and more.
-- **CSS**: A presentational language used to style the appearance of your web content (fonts, colors, layout, etc).
+- **CSS**: A presentational language used to style the appearance of your web content (fonts, colors, layout, etc). It's called Cascading Style Sheets.
```

Comment:
Good call to give context to this. Can you change it to "It stands for Cascading Style Sheets"?